src/ucs/datastruct/khash.h

```diff
@@ -649,7 +649,7 @@ typedef enum ucs_kh_put {
 		code;												\
 	} }
 
-/* More conenient interfaces */
+/* More convenient interfaces */
```

Comment:
this typo was in orig khash code
IMO better not make too many changes here so would be easier to update version if needed